docs/README.md

```diff
@@ -126,6 +126,7 @@ Metabase's reference documentation.
 - [Basic exploration](./exploration-and-organization/exploration.md)
 - [Collections](./exploration-and-organization/collections.md)
 - [History](./exploration-and-organization/history.md)
+- [Delete and restore](./exploration-and-organization/delete-and-restore.md)
```

Comment:
```suggestion
- [Trash](./exploration-and-organization/delete-and-restore.md)
```

I get that you're going for keywords `delete` and `restore`, but we can use trash here, as that's what the page covers.